INFO     backend.layers.business.business:business.py:534 {'message': 'ingesting dataset', 'collection_version_id': 5c7feef6-430d-4b7f-aeb3-1c44ee0dcb5d, 'url': 'http://fake.url', 'current_dataset_version_id': None}
INFO     backend.layers.business.business:business.py:534 {'message': 'ingesting dataset', 'collection_version_id': 5c7feef6-430d-4b7f-aeb3-1c44ee0dcb5d, 'url': None, 'current_dataset_version_id': 162228d6-27a8-410a-8605-2befbc83fb5b}
INFO     processing:dataset_metadata_update.py:319 Main: No raw h5ad update required
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:logger.py:26 Start create_artifact
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:process_logic.py:91 Uploaded [acf3b0fa-9f09-4623-abf4-dd008dbf1229/raw.h5ad] to s3://artifact_bucket/acf3b0fa-9f09-4623-abf4-dd008dbf1229/raw.h5ad
INFO     processing:process_logic.py:93 Updated database with raw_h5ad.
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:logger.py:28 Complete create_artifact
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:dataset_metadata_update.py:323 Main: Starting thread for h5ad update
INFO     processing:dataset_metadata_update.py:343 Main: Starting thread for rds update
ERROR    processing:dataset_metadata_update.py:381 Cannot find cxg artifact uri for 162228d6-27a8-410a-8605-2befbc83fb5b.
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:process_logic.py:45 Updating processing status